dll/win32/kernel32/kernel32.spec

```diff
@@ -323,8 +323,8 @@
 @ stdcall FindVolumeMountPointClose(ptr)
 @ stdcall FlsAlloc(ptr)
 @ stdcall FlsFree(long)
-@ stdcall FlsGetValue(long)
-@ stdcall FlsSetValue(long ptr)
+@ stdcall -norelay FlsGetValue(long)
+@ stdcall -norelay FlsSetValue(long ptr)
```

Comment:
How did you come by this 'fix' ?
`-norelay` is completely ignored at the moment in reactos...

Reply:
Yep, rebuild ROS again w/o -norely and all works well